atmo/templates/atmo/jobs/detail.html

```diff
@@ -163,6 +164,36 @@
         {% endfor %}
         </ul>
       </div>
+      <div role="tabpanel" class="tab-pane" id="runs">
+        {% if spark_job.has_never_run %}
+          <p>No job run history yet.</p>
+        {% else %}
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Jobflow ID</th>
+                <th>EMR version</th>
+                <th>Last status</th>
+                <th>Scheduled</th>
+                <th>Ran</th>
+                <th>Terminated</th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for run in spark_job.runs.all %}
+              <tr>
+                <td>{{ run.jobflow_id }}</a></td>
```

Comment:
It'd be nice to use the Jobflow ID to link to the AWS EMR console if the current user has permissions (is staff? member of "Spark job maintainers" group which exists in prod?).

The URL for a job is `https://<AWS region>.console.aws.amazon.com/elasticmapreduce/home?region=<AWS region>#cluster-details:<Jobflow ID>`, e.g. `https://us-west-2.console.aws.amazon.com/elasticmapreduce/home?region=us-west-2#cluster-details:j-IQROSTWRHXIH`.